lib/tasks/publishing_api.rake

```diff
@@ -0,0 +1,11 @@
+namespace :publishing_api do
+  task :republish_content => [:environment] do
+    puts "Scheduling republishing of #{Edition.published.count} editions"
+
+    Edition.published.each do |edition|
```

Comment:
If (and only if) the result set is large, and given you only need the ID perhaps you could just `Edition.published.pluck(:id).each(&blk)` to avoid having to hydrate the full `Edition`?